app/src/main/java/org/mozilla/fenix/tabstray/syncedtabs/SyncedTabsIntegration.kt

```diff
@@ -99,34 +98,4 @@ class SyncedTabsIntegration(
             )
         )
     }
-
-    /**
-     * Converts [SyncedTabsView.ErrorType] to [SyncedTabsListItem.Error] with a lambda for ONLY
-     * [SyncedTabsView.ErrorType.SYNC_UNAVAILABLE]
-     */
-    private fun SyncedTabsView.ErrorType.toSyncedTabsListItem() = when (this) {
```

Comment:
I am not sure it's worthwhile to refactor this into a `ext/` since you gotta pass in the `context` and `navController`.  

I think it was clear to me that we shouldn't refactor it when your test file ended up being called `SyncedTabsIntegrationTest` instead of `SyncedTabsViewErrorTypeTest`. So, maybe we should just make this function visible for testing.

Reply:
The naming of the test file is a mistake. I tried making the extension internal and visible for testing, but it wasn't getting imported into the test.

Reply:
Hmm.. `VisibleForTesting` not working for an `internal` function sounds a bit weird.